### PR TITLE
Mike spencer

### PR DESCRIPTION
**Pull request by pweschmidt**
episode with Mike Spencer

### CODE REVIEW

File: _posts/2020-10-01-mike-spencer.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Pushing the bubble"
+author: "@pweschmidt"
+rse: "Mike Spencer"
+phenotype: 
+excerpt: "From assessing snow fall in Britain to livestock movements, Mike has provided his expertise in data science to policy makers and influencers. So testing the data and underlying assumptions is an essential step. For this it is sometimes necessary to move outside our comfort zone."
+date: 2020-10-02 8:30:00
```
Comment (vsoch) [inline _posts/2020-10-01-mike-spencer.md:8]: I think this Thursday is October 1st?
```suggestion
date: 2020-10-01 8:30:00
```

File: _posts/2020-10-01-mike-spencer.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+duration: "00:25:39"
+explicit: "no"
+resources:
+  - name: Mike's Portfolio Blog
+    url: https://scottishsnow.wordpress.com
+  - name: Mike's Twitter
+    url: https://twitter.com/MikeRSpencer
+  - name: Mike's Webpage
+    url: http://mikerspencer.com
+    - name: Scotland's Rural College
```
Comment (vsoch) [inline _posts/2020-10-01-mike-spencer.md]: Adjusting spacing to be rendered at same level as others.

```suggestion
  - name: Scotland's Rural College
```

File: _posts/2020-10-01-mike-spencer.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    url: https://ww1.sruc.ac.uk 
+--- 
+Well before he realised he is an RSE [Mike](https://twitter.com/MikeRSpencer) was already involved in software development and data science for some time. And you can't help but be impressed with the range of projects Mike has been [working on](https://scottishsnow.wordpress.com). In this episode we focus on two of them: snow fall in Britain and livestock movement for [Scotland's Rural College](https://ww1.sruc.ac.uk). His and his colleagues' work provide data for policy influencers and makers. It is therefore important that the quality of data and underlying assumptions are properly assessed. Getting this balance right is often difficult and requires, as Mike says, "pushing the bubble".
```
Comment (vsoch) [inline _posts/2020-10-01-mike-spencer.md]:
```suggestion
Well before he realised he is an RSE [Mike](https://twitter.com/MikeRSpencer) was already involved in software development and data science for some time. And you can't help but be impressed with the range of projects Mike has been [working on](https://scottishsnow.wordpress.com). In this episode we focus on two of them: snow fall in Britain and livestock movement for [Scotland's Rural College](https://ww1.sruc.ac.uk). His and his colleagues' work provide data for policy influencers and makers. It is therefore important that the quality of data and underlying assumptions are properly assessed. Getting this balance right is often difficult and requires, as Mike says, "pushing the bubble." 
```

File: _posts/2020-10-01-mike-spencer.md (hunk 1 above)
Comment (pweschmidt) [reply _posts/2020-10-01-mike-spencer.md:8]: oops - erm, yes. So it is. Sorry, got it mixed up in the calendar

**Review by vsoch — APPROVED** [review]

**Pull request merged**